vision/unit_tests/test__gax.py

```diff
@@ -32,12 +37,55 @@ def test_ctor(self):
             api = self._make_one(client)
         self.assertIs(api._client, client)
 
+    def test_gapic_credentials(self):
+        from google.cloud.gapic.vision.v1.image_annotator_client import (
+            ImageAnnotatorClient)
+        from google.cloud.vision import Client
+
+        # Mock the GAPIC ImageAnnotatorClient, whose arguments we
+        # want to check.
+        with mock.patch.object(ImageAnnotatorClient, '__init__') as iac:
+            iac.return_value = None
+
+            # Create the GAX client.
+            credentials = _make_credentials()
+            client = Client(credentials=credentials)
```

Comment:
The build is failing when trying to [determine the project](https://travis-ci.org/GoogleCloudPlatform/google-cloud-python/builds/204324421#L870).

I think you'll have to set `project` when you create the client. `client = Client(project='my-project', credentials=credentials)`